plugins/ftdidmx/FtdiDmxThread.cpp

```diff
@@ -25,31 +25,51 @@
 
 #include <math.h>
 #include <unistd.h>
+#include <time.h>
 
 #include <string>
+#include <queue>
+#include <utility>
 
 #include "ola/Clock.h"
 #include "ola/Logging.h"
 #include "ola/StringUtils.h"
+#include "ola/Constants.h"
+
+#include "ola/rdm/RDMCommand.h"
+#include "ola/rdm/RDMControllerInterface.h"
+#include "ola/rdm/RDMCommandSerializer.h"
+#include "ola/rdm/RDMResponseCodes.h"
+#include "ola/rdm/DiscoveryAgent.h"
+
 #include "plugins/ftdidmx/FtdiWidget.h"
 #include "plugins/ftdidmx/FtdiDmxThread.h"
 
 namespace ola {
 namespace plugin {
 namespace ftdidmx {
 
-FtdiDmxThread::FtdiDmxThread(FtdiInterface *interface, unsigned int frequency)
+FtdiDmxThread::FtdiDmxThread(FtdiInterface *interface,
+                             unsigned int frequency,
+                             unsigned int serial)
   : m_granularity(UNKNOWN),
     m_interface(interface),
     m_term(false),
-    m_frequency(frequency) {
+    m_frequency(frequency),
+    m_transaction_number(0),
```

Comment:
I think this is now managed centrally due to what I did here: https://github.com/OpenLightingProject/ola/pull/1343/files

Reply:
Does this mean I can remove all code and references to this?

At the moment for this pull I tend to think leave it and then I can remove and test again, but whatever you prefer.